packages/datadog-plugin-aws-sdk/src/services/bedrockruntime/utils.js

```diff
@@ -234,17 +236,45 @@ function extractRequestParams (params, provider) {
       })
     }
     case PROVIDER.AMAZON: {
+      const prompt = requestBody.inputText
       if (modelId.includes('embed')) {
-        return new RequestParams({ prompt: requestBody.inputText })
+        return new RequestParams({ prompt })
+      } else if (prompt !== undefined) {
```

Comment:
```suggestion
      } else if (prompt) {
```

unless `prompt` can be an empty string while `requestBody.messages` still exists, i think we can just do this check

Reply:
I think prompt can be an empty string and we might want to still capture the other request params in this case which we wouldn't do if we jumped all the way down to `return new RequestParams({ prompt })`, so I may leave it as is.